tools/chapel-py/src/core-types.h

```diff
@@ -56,6 +56,8 @@ struct ScopeObject : public PythonClassWithObject<ScopeObject, const chpl::resol
   static constexpr const char* DocStr = "A scope in the Chapel program, such as a block.";
 };
 
+using VisibileSymbols = std::vector<std::tuple<chpl::UniqueString, std::vector<const chpl::uast::AstNode*>>>;
```

Comment:
I think this would look cleaner as
```
using VisibileSymbol = std::tuple<chpl::UniqueString, std::vector<const chpl::uast::AstNode*>>;
using VisibileSymbols = std::vector<VisibileSymbol>;
```
At this point, its probably not even necessary to have `using VisibileSymbols = ...`